src/Tests/dotnet-new.Tests/TemplateEngineSamplesTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.NET.TestFramework;
+using Microsoft.TemplateEngine.Authoring.TemplateVerifier;
+using Microsoft.TemplateEngine.TestHelper;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Cli.New.IntegrationTests
+{
+    [Collection("Verify Tests")]
+    public class TemplateEngineSamplesTest : BaseIntegrationTest, IClassFixture<SharedHomeDirectory>
+    {
+        private readonly SharedHomeDirectory _sharedHome;
+        private readonly ILogger _log;
+
+        public TemplateEngineSamplesTest(SharedHomeDirectory sharedHome, ITestOutputHelper log)
+            : base(log)
+        {
+            _log = new XunitLoggerProvider(log).CreateLogger("TestRun");
+            _sharedHome = sharedHome;
+            _sharedHome.InstallPackage(
+               "Microsoft.TemplateEngine.Samples",
+               nugetSource: @"https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json");
```

Comment:
Can we omit the explicit nuget source and let the `dotnet new install` use the sources from nuget.config in repo root?